.github/workflows/notify-slack.yml

```
@@ -0,0 +1,77 @@
+---
+
+name: 'Notify slack'
+
+on:
+  workflow_call:
+    inputs:
+      application:
+        description: 'The friendly name for the application'
+        type: string
+        required: true
+
+      environment:
+        description: |
+          The environment variables to pass to the render.  Each environment
+          variable should be in format VAR=VAL with one on each line.
+          Example:
+            environment: |
+              FOO=bar
+              BAR=baz
+        type: string
+        required: false
+        default: ''
+
+      undefined:
+        description: 'If true, undefined variables will be used in templates (no error will be raised)'
+        type: string
+        required: false
+        default: 'false'
+
+      template:
+        description: 'The path to the template file to render'
+        type: string
+        required: true
+
+      version:
+        description: 'The version that is being deployed'
+        type: string
+        default: ${{ github.ref }}
+
+      type:
+        description: 'The type of action that is being performed such as build or deployment'
+        type: string
+        required: true
+
+      status:
+        description: 'The status of the deployment.  Options are started, success, and failure'
+        type: string
+        required: true
+
+    secrets:
+      slack_webhook_url:
```

Let's please stick with the convention we have established with other reusable workflows/actions which is to use dashes instead of underscores.

```suggestion
      slack-webhook-url:
```

---

As a note, this secret is unused within this workflow.